### PR TITLE
Updates to Apply Form

### PR DESCRIPTION
**Pull request by xtine**
Using the latest data in the Google Form.

### CODE REVIEW

File: assets/css/form.css [hunk 1]
```diff
@@ -47,10 +47,26 @@ header:after {
   visibility: hidden;
 }
 
+.required {
```
Comment (fureigh) [inline assets/css/form.css:50]: @xtine Were these changes intentional? Asking because they're not content changes, not because they don't look good. :)

File: pages/apply.html [hunk 2]
```diff
@@ -79,103 +82,67 @@
...
 
 
-      <fieldset>
+      <fieldset class="veteran-status">
 
-      <h2><legend>Eligibility:</legend></h2>
+        <h2><legend>Veteran Status:</legend></h2>
 
-        <p>
-          Any individual who is not currently a GSA employee or contractor and who is a U.S. Citizen, a non-citizen national of the U.S., or has been admitted to the U.S. for permanent residence and holds a valid green card is eligible to apply.
```
Comment (fureigh) [inline pages/apply.html:87]: @rooneywp Is this still the accurate language? If so, we've lost it in the current version, which asks only for verification that the applicant is a U.S. Citizen or Green Card holder, and we should restore it.

File: assets/css/form.css (hunk 1 above)
Comment (xtine) [inline assets/css/form.css:50]: Yeah, I wanted to better highlight the required fields.

File: pages/apply.html [hunk 3]
```diff
@@ -211,50 +182,79 @@
...
-              <label for="group_305267568_6">10-point preference for non-compensable disability or Purple Heart (XP).</label>
-            </li>
-            <li>
-              <input type="radio" name="entry.305267568" value="10-point preference based on widow/widower or mother of a deceased veteran, or spouse or mother of a disabled veteran (XP)." id="group_305267568_7" role="radio" class="ss-q-radio" aria-label="10-point preference based on widow/widower or mother of a deceased veteran, or spouse or mother of a disabled veteran (XP)." aria-required="true">
-              <label for="group_305267568_7">10-point preference based on widow/widower or mother of a deceased veteran, or spouse or mother of a disabled veteran (XP).</label>
-            </li>
-          </ul>
+          <p>Please see &quot;Government pay grades explained:&quot; <a href="https://www.google.com/url?q=https://pages.18f.gov/joining-18f/pay-grades/&amp;sa=D&amp;usg=AFQjCNHPdTF2o1CKuVPZtq_dRjd8jNlsvw">https://pages.18f.gov/joining-18f/pay-grades/</a></p>
```
Comment (fureigh) [inline pages/apply.html]: This should be a direct link instead of a link to a Google result for this link. We can actually just link the words 'Government pay grades explained' now that we have access to the magic of HTML. :rocket:

File: assets/css/form.css (hunk 1 above)
Comment (fureigh) [inline assets/css/form.css:50]: Cool. Thanks for doing that!

File: pages/apply.html [hunk 4]
```diff
@@ -263,21 +263,65 @@
...
+
+          <label class="ss-q-item-label" for="entry_510169774">Why do you want to join 18F?</label>
+          <textarea name="entry.510169774" rows="8" cols="0" class="ss-q-long" id="entry_510169774" dir="auto" aria-label="Why do you want to join 18F?  "></textarea>
+
         </div>
 
+        <div class="usa-input">
+
+          <label class="ss-q-item-label" for="entry_1108678713">What role or team at 18F do you believe you&#39;re the best fit for and why?</label>
+          <p>If you need inspiration, check out this page: <a href="https://www.google.com/url?q=https://pages.18f.gov/joining-18f/who-we-are-hiring/&amp;sa=D&amp;usg=AFQjCNF6V5k3-x9thBTB3cgwRzDJf9M6tQ">https://pages.18f.gov/joining-18f/who-we-are-hiring/</a>. If you don&#39;t see a perfect role there, that&#39;s ok! Just tell us what you think.</p>
```
Comment (fureigh) [inline pages/apply.html]: Same comment as on line 203.

File: pages/apply.html [hunk 5]
```diff
@@ -263,21 +263,65 @@
...
+
+        <h2><legend>Résumé:</legend></h2>
+
         <div class="usa-input">
-          <label for="entry_422263110">Where did you learn about 18F?</label>
-          <textarea name="entry.422263110" rows="8" cols="0" class="ss-q-long" id="entry_422263110" dir="auto" aria-label="Where did you learn about 18F?  "></textarea>
+
+          <label class="ss-q-item-label" for="entry_710467749">Résumé <span class="required">*</span></label>
+
+          <p>Please copy and paste your résumé (or a link to it) below. Do NOT include your Social Security Number. Please be aware that cutting and pasting will remove most formatting. Be sure to include the number of years of experience for any given position.</p>
```
Comment (fureigh) [inline pages/apply.html]: Let's change each `é` to an `&eacute;` to be on the safe side.
